search-app/resources/temporal_overlap.groovy

```diff
@@ -1,26 +1,35 @@
 /* Elastic groovy script to calculate the temporal overlap of a collection over the input ranges.
-   Collection start and end dates are doc['start-date'] and doc['end-date'].
+
+   Collection temporal ranges are in doc['temporal-ranges'], which is a list of dates with no
+   overlaps. When sorted, they will be in the order: start-date, end-date, start-date, end-date...
+   If the list has an odd number of elements, the collection is ongoing and has no end date.
```

Comment:
I thought we always have a nil for the end-date, so there won't be odd number of elements?

Reply:
Elastic will remove the nils, so when the list comes through the script, there will be no nils. You can play around with Marvel to see.

Reply:
Thanks.